composer/trainer/checkpoint.py

```diff
@@ -335,7 +335,7 @@ class CheckpointSaver:
         interval (Time or str): The amount of time units to wait between checkpoints.
     """
 
-    def __init__(self, save_folder: str, interval: Union[Time, str]):
+    def __init__(self, save_folder: str, interval: Union[Time, str], compression: str):
```

Comment:
```suggestion
    def __init__(self, save_folder: str, interval: Union[Time, str], compression: str = ""):
```

Also docstring update

Reply:
Going to commit the suggestion (pretty innocent), and then merge with red button (no need to rerun all of jenkins for this one change)